File: las-notes-mac-win-native/macos/LasNotes/model/Note.swift
/// Case class to represent a note
/// - Parameters:
///   - id: note ID
///   - data: markdown string
///   - tags: comma-separated tags
///   - isDeleted: soft-deleted marker
struct Note: Identifiable {
    let id: Int64
    let data: String
    let tags: String
    let isDeleted: Bool
}

File: las-notes-mac-win-native/windows/LasNotes/model/Note.cs
namespace LasNotes;

/// <summary>
/// Case class to represent a note
/// </summary>
/// <param name="Id">note ID</param>
/// <param name="Data">markdown string</param>
/// <param name="Tags">comma-separated tags</param>
/// <param name="IsDeleted">soft-deleted marker</param>
internal record Note(long Id, string Data, string Tags, bool IsDeleted);
